elements-code-tutorial/advanced-examples.md

```diff
@@ -105,11 +105,17 @@ fi
 
 shopt -s expand_aliases
 
-alias e1-dae="$HOME/elements/src/elementsd -datadir=$HOME/elementsdir1"
-alias e1-cli="$HOME/elements/src/elements-cli -datadir=$HOME/elementsdir1"
+# Before running this script, change the following variables to point to: your local Elements binaries directory, the directory you'll use to store node data (note that the script will delete the $DATA_DIR/elementsregtest directory when run, so back up anything you might already have in there before running).
+
+ELEMENTS_DIR="$HOME/elements"
```

Comment:
I think if we're doing this we might as well include the src part as well. I currently use the binary release so I'd have to amend this and line 114 etc if I used this. So add src path to this and then have line 114 and 115 just add the elementsd and elements-cli bits.

Reply:
I originally had it that way but something was also needed further down when copying the config files:
```
cp $ELEMENTS_DIR/contrib/assets_tutorial/elements1.conf $DATA_DIR1/elements.conf
cp $ELEMENTS_DIR/contrib/assets_tutorial/elements2.conf $DATA_DIR2/elements.conf
```
I can add a BINDIR that's $ELEMENTS_DIR/src.

I also see another change I should have made related to this. I'll push and update.